src/main/java/br/ufpe/cin/mergers/handlers/ConflictHandler.java

```diff
@@ -0,0 +1,13 @@
+package br.ufpe.cin.mergers.handlers;
+
+import br.ufpe.cin.exceptions.TextualMergeException;
+import br.ufpe.cin.mergers.util.MergeContext;
+
+/**
+ * Interface responsbile for dealing with language specific conflicts that
+ * just superimposition of trees is not able to detect/resolve.
+ * @author Guilherme
+ */
+public interface ConflictHandler {
+	void handle(MergeContext context) throws TextualMergeException;
```

Comment:
we should probably rename the Exception

Reply:
not really, one of the handlers calls TextualMerge, which might throw that exception. 

TextualMergeException is the most critical exception for us because it represents inability to give an output. For instance, if the parser fail's, TextualMerge is called as fallback, but if TextualMerge fails too, the user will not have an output, and the tool crashes. (see https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/app/JFSTMerge.java#L178)

Reply:
@guilhermejccavalcanti if we are trying to create an abstraction by defining an interface then, yes, we should rename the exception. the abstraction cannot rely on the details of one of its particular implementations. we should then have a ConflictHandlingException

Reply:
I disagree about renaming that exception. But if are talking about chaining the exception, to throw again as ConflictHandlingException, OK given that will respect the critical aspect I mentioned before about textual merge.

Reply:
Making it simple, if TextualMergeExecption happens the tool should crashe, log the error, and notify the user. So a ConflictHandlingException must have the same consequence if it chains TextualMergeExecption.

Reply:
yes, that makes more sense. however, we should not do this for all instances of ConflictHandlingException, but just for its instances (the ones that encapsulate TextualMergeExecption)

Reply:
@guilhermejccavalcanti could you please point out where TextualMergeExecption (the one raised by the handler) is handled?

Reply:
Here https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/app/JFSTMerge.java#L176

Reply:
OK, then what we discussed makes sense